spec/features/bulk_edit_spec.rb

```diff
@@ -57,9 +57,9 @@
     it "updates the object" do
       visit bulk_edit_resources_edit_path("q" => "", "f[member_of_collection_titles_ssim][]" => "My Collection")
       expect(page).to have_content "You searched for"
-      page.check("mark_complete")
+      page.driver.execute_script("document.getElementById('mark_complete').click()")
```

Comment:
I found this necessary given that I could not manage to resolve this exclusively on CircleCI (this passed for me locally, and I did attempt to ensure that the Node.js, Google Chrome, and chromedriver on CircleCI matched those releases which I used locally).